pkg/conditions/conditions.go

```diff
@@ -0,0 +1,56 @@
+package conditions
```

Comment:
Shouldn't this be `condition`?

Reply:
Sure, as this package is centered around the OperatorCondition CR.